### PR TITLE
Add Sponsors to the API

### LINKED ISSUE #308: API should support sponsorship info
To work with PyGotham sponsorship information programmatically, we need access to the data. The API should have a new endpoint (`/sponsors/`) that exposes the following information (everything available on the sponsors page):
- Name
- Level
- URL
- Description
- Logo URL

### PR DESCRIPTION
**Pull request by jonafato**
Fixes #308.

### CODE REVIEW

File: pygotham/api/sponsors.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+api = Api(blueprint)
+
+
+@api.resource('/')
+class SponsorResource(Resource):
+    """Represents event sponsors."""
+
+    @marshal_with(sponsor_fields)
+    def get(self, event_id):
+        """Return a list of accepted talks."""
```
Comment (dirn) [inline pygotham/api/sponsors.py]: I think this is a copy and paste oops.

File: pygotham/api/sponsors.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+"""Sponsor-related API endpoints."""
+
+from flask import Blueprint
+from flask_restful import Api, Resource
+
+from pygotham.events.models import Event
```
Comment (dirn) [inline pygotham/api/sponsors.py]: Is there a reason not to use `from pygotham.models import Event, Level, Sponsor`?

Comment (jonafato) [inline pygotham/api/sponsors.py]: No particular reason. This was based on the schedule API module, which imports `Event` and `Talk` separately. I'm open to changing it; we should standardize on one approach.

Comment (dirn) [inline pygotham/api/sponsors.py]: We should, but that's probably a topic for a separate pull request.

**Pull request merged**